### ISSUE TITLE
Read UUIDs seem scrambled when per-read modified sqlite database gets split by motif

### ISSUE THREAD

**Issue by AAnnan** [nanoporetech/megalodon#54]
Hello,

I'm trying to extract single read information from the per-read modified sqlite database. It works fine on the whole `per_read_modified_base_calls.db` output by Megalodon. 

However after using `megalodon_extras modified_bases split_by_motif` to split the database into motif-specific databases, I found that the UUIDs listed in the subset databases and their associated scores and position do not match with the ones in the original database. Nor do they match with information present in `mappings.summary.txt`. It seems the read_ids get scrambled when the database gets split.

I use the same sqlite command for both whole and split databases:
```
SELECT (score_pos/2),score 
FROM data 
INNER JOIN read ON data.score_read = read.read_id 
WHERE read.uuid = '{read_id}' AND data.score_pos%2 = {strand}
 ```

**Comment by marcus1487** [comment]
This does indeed appear to be a bug in the `split_by_motif` function. A fix for this should be released soon.

**Comment by marcus1487** [comment]
This was fortunately a simple fix (a0e4a4c). Please re-open this issue if this issue persists. 

P.S. I'm interested in making per-read analyses easier within the Megalodon framework. Is there a particular access pattern that would make your analysis easier?
